contracts/sol6/fprConversionRates/SimpleVolumeImbalanceRecorder.sol

```diff
@@ -0,0 +1,162 @@
+pragma solidity 0.6.6;
+
+import "@kyber.network/utils-sc/contracts/Withdrawable.sol";
+
+/// @dev this is another verison of SignedSafeMath(OpenZeppelin) for int64
+library SafeInt64 {
```

Comment:
consider to just put it inside the contract as an internal func

Reply:
- This is a library from SignedSafeMath, so I want to put in a different module
- For easy, type cast
 ```using SafeInt64 for int64;```

Reply:
Should we move it to another file?

Reply:
Ok